### PR TITLE
docs: update devops

### PR DESCRIPTION
**Pull request by raisedadead**

### CODE REVIEW

File: docs/devops.md [hunk 1]
```diff
@@ -7,82 +7,47 @@
...
+We employ various levels of integration and acceptance testing to check on the quality of the code. All our tests are done through software like Travis CI, Circle CI and Azure Pipelines.
 
-## Identifying the current version of platform
+We have unit tests for testing our challenge solutions, Server APIs and Client User interfaces. These help us test the integration between different components.
 
-**The current version of the platform is always available at [`freeCodeCamp.org`](https://www.freecodecamp.org).**
+We are also in the process of writing end user tests which will help in replicating real world scenarios like updating an email or making a call to the API or third-party services.
 
-The dev-team merges changes from the `production-staging` branch to `production-current` when they release changes. The top commit should be what you see live on the site. You can identify the exact version deployed by visiting the build and deployment logs available below in the status section.
+These tests, help in preventing issues from repeating themselves and ensure that while fixing an issue or adding a feature, we do not introduce a regression of some form.
```
Comment (RandellDawson) [inline docs/devops.md]:
```suggestion
These tests help in preventing issues from repeating themselves and ensure that while fixing an issue or adding a feature, we do not introduce a regression of some form.
```

**Review by RandellDawson — CHANGES_REQUESTED** [review]
One small suggestion.

File: docs/devops.md (hunk 1 above)
Comment (raisedadead) [reply docs/devops.md]: Sure, I will update this. I am still adding/moving a bunch of things. The idea is soon anyone on the mods team will be able to push .dev :)

Comment (raisedadead) [reply docs/devops.md]: Sorry, I could not add changes that would have enabled the mods team to push builds yet. I will get it done maybe by next week or so.

**Pull request merged**